blog/2024/09-23-version-1.0/index.mdx

```diff
@@ -0,0 +1,83 @@
+For easier management, you can organize users into groups and manage them all at the same time.
+- **Integration testing**: Starting from ``1.0``, we will require you to test the integration before you can use it.
+This guarantees that you've configured your integration correctly and helps you find errors faster.
+- **Re-imagined widgets**: All widgets have been rewritten & improved. They offer much better performance,
+reliability and scalability compared to < ``1.0``.
+- **Improved performance**: We have upgraded the underlying framework of Homarr and take advantage
+of an in-memory message bus and websockets to deliver data near realtime to you.
+
+<!-- truncate -->
+
+## Breaking changes
+:::warning
+
+Please consult the breaking changes list & upgrade guide if you used third-party installation methods.
+
+:::
+
+- Docker Image
+  - Removed support for ``arm/v7`` -> upgrade to newer architectures as an alternative
+  - Renamed image from ``ghcr.io/ajnart/homarr`` to ``ghcr.io/homarr-labs/homarr``. We now use semantic versioning for our release numbers.
+- Environment Variables
+  - Removed ``DISABLE_ANALYTICS``, can now be configured via UI
+  - Renamed ``AUTH_PROVIDER`` to ``AUTH_PROVIDERS``
+  - Changed default value of ``PORT`` from ``7575`` to ``3000``
+  - Renamed ``DATABASE_URL`` to ``DB_URL`` and made it optional conditionally. If ``DB_HOST`` is set, this is optional - otherwise it's required.
+  - Added ``DB_DRIVER`` with values ``better-sqlite3`` (default) or ``mysql2``.
+  - Added other database configuration environment variables: ``DB_URL``, ``DB_PORT``, ``DB_USER``, ``DB_PASSWORD``, ``DB_NAME`` (consult documentation for detailed information on these)
+  - Changed ``DOCKER_HOST`` from a single string to a comma seperated string list
+  - ``AUTH_LOGOUT_REDIRECT_URL`` must now be a valid Url
+  - Increased minimum length of ``AUTH_OIDC_CLIENT_ID`` to 1
+  - Increased minimum length of ``AUTH_OIDC_CLIENT_SECRET`` to 1
+  - Removed ``AUTH_OIDC_TIMEOUT`` **<ins># TODO: Discuss whether we need to copy this?</ins>**
+  - Removed ``AUTH_OIDC_AUTO_LOGIN`` **<ins># TODO: Discuss whether we need to copy this?</ins>**
+  - Removed ``AUTH_LDAP_ADMIN_GROUP`` and ``AUTH_LDAP_OWNER_GROUP`` -> groups are now managed without manual env binding
+  - ``CI`` will now also trigger an env validation skip like ``SKIP_ENV_VALIDATION``
+- Mounts
+  - Removed ``/app/data/configs`` -> JSON configs have been removed in favour of relation databases
+  - Changed ``/app/public/icons`` to ``/images`` **<ins># TODO: Do we want this path? Is the local source already working?</ins>**
+  - Changed structure of ``/data`` -> Create a backup before migration of the entire directory to copy data to new Homarr
+- Database:
+  - Changed schema of the database to a completely new schema. They are not backwards compatible. Please export the data from old Homarr in the UI and import it in ``1.0``
```

Comment:
Users and such can not be imported, just to make this clear

Reply:
Yes, I know. But we should make them import-able, no? We have the hashed password and we should be able to copy that, correct?

Reply:
Yes thats true and would be possible, salt has to be copied as well btw